skin/src/main/java/org/researchstack/skin/DataProvider.java

```diff
@@ -105,7 +106,7 @@ public static void init(DataProvider instance)
      * @return Observable of the result of the method, with {@link DataResponse#isSuccess()}
      * returning true if signIn was successful
      */
-    public abstract Observable<DataResponse> resendEmailVerification(Context context, String email);
```

Comment:
actually adding another method doesn't make much sense, as one would have to implement both methods. I suggest that we keep the method with more parameters in, and let decide to the implementation if all parameters are actually used.